packages/aws-cdk-lib/aws-lambda/test/logging-config.test.ts

```diff
@@ -258,3 +280,33 @@ test('Throws when loggingFormat and logFormat are both specified', () => {
     });
   }).toThrow(/Only define LogFormat or LoggingFormat, not both./);
 });
+
+test('Throws when applicationLogLevel and applicationLogLevelV2 are both specified', () => {
+  const app = new cdk.App();
+  const stack = new cdk.Stack(app, 'stack');
+  expect(() => {
+    new lambda.Function(stack, 'Lambda', {
+      code: new lambda.InlineCode('foo'),
+      handler: 'index.handler',
+      runtime: lambda.Runtime.NODEJS_18_X,
+      loggingFormat: lambda.LoggingFormat.JSON,
+      applicationLogLevel: lambda.ApplicationLogLevel.INFO,
+      applicationLogLevelV2: lambda.ApplicationLogLevel.WARN,
+    });
+  }).toThrow(/Only define applicationLogLevel or applicationLogLevelV2, not both./);
+});
+
+test('Throws when systemLogLevel and systemLogLevelV2 are both specified', () => {
+  const app = new cdk.App();
+  const stack = new cdk.Stack(app, 'stack');
+  expect(() => {
+    new lambda.Function(stack, 'Lambda', {
+      code: new lambda.InlineCode('foo'),
+      handler: 'index.handler',
+      runtime: lambda.Runtime.NODEJS_18_X,
+      loggingFormat: lambda.LoggingFormat.JSON,
+      systemLogLevel: lambda.SystemLogLevel.INFO,
+      systemLogLevelV2: lambda.SystemLogLevel.WARN,
+    });
+  }).toThrow(/Only define systemLogLevel or systemLogLevelV2, not both./);
+});
```

Comment:
Can you add one more test for the case where `systemLogLevelV2` or `applicationLogLevelV2` is set but `loggingFormat` isn't set to `LoggingFormat.JSON`